src/main/java/zaasclient/ConfigProperties.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package zaasclient;
+
+public class ConfigProperties {
+
+    private String apimlHost;
+    private String apimlPort;
+    private String apimlBaseUrl;
+    private String keyStoreType;
+    private String keyStorePath;
+    private String keyStorePassword;
```

Comment:
We should declare passwords as an array of chars (char[]) instead of String. 
String is immutable,  while char[] allows us to clear the password when we finish with it.